### DISCUSSION TITLE
PermutationLayout of TiledMMA

### DISCUSSION THREAD

**Discussion by yangjianfengo1** [NVIDIA/cutlass#1419] [Q&A]
my TiledMMA is 

```
TiledMMA<
        MMA_Atom<SM80_16x8x16_S32S8S8S32_TN>,
        Layout<Shape<_1,_1,_1>>, 
        Tile<_32,_32,_32>>;
```

layout is 

<img width="756" alt="image" src="https://github.com/NVIDIA/cutlass/assets/125249383/c8d2874c-dc43-4892-8f29-edfd1c0cd8ac">

I want the elements of matrix C to be consecutive, for instance, having the four elements (T0,V0), (T0,V1), (T0,V8), and (T0,V9) together. so，I modified TiledMMA to

```
TiledMMA<
        MMA_Atom<SM80_16x8x16_S32S8S8S32_TN>,
        Layout<Shape<_1,_1,_1>>, 
        Tile<_32,
           Layout<Shape <_2,_4,_4>, 
                                    Stride<_1,_4,_4>>,
          _32>>;
```

But this operation will cause compilation errors. Who can help me get a correct PermutationLayout? I would appreciate it.

**Comment by thakkarV** [comment]
can you please provide a bit info about the context in which you are using this tiled MMA? Why are you setting the tiled MMA K dimension to be 32 and setting the tile to be so large without changing the atom layout?
